test/viewmodels/PaymentSearchResultStatementsViewModelSpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import config.AppConfig
+import play.api.Application
+import play.api.i18n.Messages
+import utils.SpecBase
+import utils.TestData.*
+
+class PaymentSearchResultStatementsViewModelSpec extends SpecBase {
+
+  "apply" should {
+
+    "return viewModel object with correct content" when {
+
+      "cash transactions are available" in new Setup {
+        val viewModel01: PaymentSearchResultStatementsViewModel =
+          PaymentSearchResultStatementsViewModel(SEQ_OF_PAYMENT_DETAILS_01, None)
+
+        viewModel01.dailyStatements.size mustBe 3
```

Comment:
Could you please add an assertion for the dailyStatements values along with this size assertion? Leave it if it takes time for you